runtime/makelib/targets.mk.zos.inc.ftl

```diff
@@ -68,7 +68,7 @@ ifdef j9vm_uma_supportsIpv6
   UMA_ZOS_FLAGS += -DIPv6_FUNCTION_SUPPORT
 endif
 
-UMA_ZOS_FLAGS += -DJ9ZOS390 -DLONGLONG -DJ9VM_TIERED_CODE_CACHE -D_ALL_SOURCE -D_XOPEN_SOURCE_EXTENDED -DIBM_ATOE -D_POSIX_SOURCE
+UMA_ZOS_FLAGS += -DJ9ZOS390 -DLONGLONG -DJ9VM_TIERED_CODE_CACHE -D_ALL_SOURCE -D_XOPEN_SOURCE_EXTENDED -DIBM_ATOE -D_POSIX_SOURCE -D_ISOC99_SOURCE
```

Comment:
This doesn't seem sufficient or complete as there are several other references to `c89` on zOS:
https://github.com/eclipse/openj9/blob/d1eb4ed8225550c810bf4b6a7e9d84f0b10b63e2/runtime/gc_glue_java/configure_includes/configure_zos_390.mk.ftl#L72-L73

https://github.com/eclipse/openj9/blob/2f295ea91cdd97ff37eb5d164fc63014ea718bc7/runtime/ddr/autoblob_rules.mk#L46 (several in this file)

https://github.com/eclipse/openj9/blob/fbbe1cd6b3fadbddf979d30239ff3dbf409b5e44/buildspecs/zos_390-64.spec#L88-L89 (same in other z-specs)

Reply:
Please update the commit to have both flags and add a comment to the UMA file describing why we need these flags.  When we update to a c99 compiler, it will be easier to find and remove these flags then